lib/rubocop/cop/rails/not_null_column.rb

```diff
@@ -44,17 +44,20 @@ def on_send(node)
         private
 
         def check_add_column(node)
-          pairs = add_not_null_column?(node)
-          check_pairs(pairs)
+          add_not_null_column?(node) do |type, pairs|
+            return if type.source == ':virtual' || type.source == "'virtual'"
```

Comment:
`type.value` should be used instead of `type.source` to detect double quoted `"virtual"`.

```suggestion
            return if type.value == :virtual || type.value == 'virtual'
```

Reply:
🤦 Thanks! Fixed.